Access API/AccessAPI/DTOs/SearchResultsDTO.cs

```diff
@@ -1,12 +1,9 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using System.Collections.Generic;
 
 namespace Access_API
 {
     public class SearchResultsDTO
     {
-        public List<SearchResultDTO> result;
+        public List<SearchResultDTO> Result;
```

Comment:
Check warning siger "Non-nullable field 'Result' must contain a non-null value when exiting constructor. Consider declaring the field as nullable."